app/components/collection_metadata_component.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="show-metadata">
-  <% if current_staff_user? %>
+  <% if can?(:read, Asset) && can?(:read, Work) %>
```

Comment:
looks like we no longer use `current_staff_user?` anywhere... is there a method defined somewhere that needs to be removed too?

Reply:
Yup - with the other branch we were working on merged, we can in fact get rid of it.